tests/integration/test_record.py

```diff
@@ -45,85 +45,50 @@ def test_incoming_deleted_record(app):
                '</record>'
                '</collection>')
 
-    json_record = clean_record(hepnames.do(create_record(snippet)))
 
+def test_incoming_deleted_record(app):
+    json_record = clean_record(hepnames.do(create_record(snippet)))
```

Comment:
Please rebase on top of latest master. @jacquerie has amended all calls to `clean_record`.